resources/js/components/requests/requestModal.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+  <b-modal
+    id="requests-modal"
+    ref="requestModalAdd"
+    class="requests-modal modal-dialog-scrollable"
+    header-close-content="&times;"
+    hide-footer
+    :title="$t('New Request')"
+    :size="size"
+  >
+    <b-row no-gutters>
+      <b-col
+        cols="12"
+        md="8"
+      >
+        <p>
+          {{ $t("We've made it easy for you to start a Request for the following Processes. Select a Process to start your Request.") }}
+        </p>
+      </b-col>
+      <b-col
+        cols="12"
+        md="4"
+      >
+        <b-input-group class="search">
+          <b-form-input
+            v-model="filter"
+            :placeholder="$t('Search') + '...'"
+          />
+          <b-input-group-append>
+            <button
+              type="button"
+              class="btn btn-primary"
+              :aria-label="$t('Search')"
+            >
+              <i class="fas fa-search" />
+            </button>
+          </b-input-group-append>
+        </b-input-group>
+      </b-col>
+    </b-row>
+
+    <div
+      v-if="Object.keys(processes).length && !loading"
+      class="process-list"
+    >
+      <div
+        v-for="(category, index) in processes"
+        :key="index"
+        class="mt-3"
+      >
+        <h5 class="mb-n2">
+          {{ index }}
+          <span class="badge badge-pill badge-secondary">
+            {{ category.length }}
+          </span>
+        </h5>
+        <template v-for="(process, id) in category">
+          <process-card
+            v-if="hasEmptyStartEvents(process)"
+            :key="id"
+            :filter="filter"
+            :process="process"
+          />
+        </template>
+      </div>
+    </div>
+
+    <div
+      v-if="!Object.keys(processes).length && !loading"
+      class="no-requests my-3 text-center"
+    >
+      <h4>{{ $t('You don\'t have any Processes.') }}</h4>
+      <span v-if="permission.includes('create-processes')">
+        <a
+          :href="url"
+          class="text-primary"
+        >
+          {{ $t('Please visit the Processes page') }}
+        </a>
+        {{ $t('and click on +Process to get started.') }}
```

Comment:
Please verify all your label are present in the translations files

Reply:
this is old code, we only move a other file in order to use in web and mobile